src/utils/utils.go

```diff
@@ -41,6 +42,24 @@ func InitLogger(verbose bool) *logrus.Logger {
 	return log
 }
 
+type CoreosInstallerLogWriter struct {
+	progressRegex *regexp.Regexp
+	log           *logrus.Logger
+	lastLogLine   []byte
```

Comment:
I don't see that this is used any where

Reply:
You are right, it's for the next task (reporting progress of the coreos-installer) I'll remove it for now.